README.md

```diff
@@ -28,6 +28,15 @@ Questions, bug reporting and issue tracking are provided by GitHub. Please
 report all bugs by creating a new issue with the bug tag. You can ask
 questions by creating a new issue with the question tag.
 
+Contributing
+------------
+
+Contributing to DTK is easy. Just send us a [pull request](https://help.github.com/articles/using-pull-requests/).
```

Comment:
@Rombur I understand your point. I would just argue that the barrier to entry should be as low as possible when the number of contributors is limited. I would be willing to work with a contributor on the test, rather than requiring it from scratch. I would not merge PR without a test.